modules/system/console/MixList.php

```diff
@@ -54,9 +50,15 @@ public function handle(): int
             }
         }
 
-        $this->table(['Name', 'Active', 'Path', 'Configuration'], $rows);
-
-        $this->line('');
+        if($this->option('json')) {
+            echo json_encode($rows, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);
+        } else {
+            $this->line('');
+            $this->info('Packages registered:');
+            $this->line('');
+            $this->table(['Name', 'Active', 'Path', 'Configuration'], $rows);
```

Comment:
Following from above, we can apply the Yes / No styling directly here:

```suggestion
            $this->table(['Name', 'Active', 'Path', 'Configuration'], array_map(function ($row) {
                $row['active'] = ($row['active']) ? '<info>Yes</info>' : '<fg=red>No</>';
                return $row;
            }, $rows));
```